src/dmd/e2ir.d

```diff
@@ -949,7 +949,8 @@ Lagain:
                     r = (op == TOK.construct) ? RTLSYM_ARRAYSETCTOR : RTLSYM_ARRAYSETASSIGN;
                     evalue = el_una(OPaddr, TYnptr, evalue);
                     // This is a hack so we can call postblits on const/immutable objects.
-                    elem *eti = getTypeInfo(tb.unSharedOf().mutableOf(), irs);
+                    AggregateDeclaration ad = cast(AggregateDeclaration)tb;
+                    elem *eti = getTypeInfo(ad.loc, tb.unSharedOf().mutableOf(), irs);
```

Comment:
This location is the one of the aggregate declaration, not the using expression. For completeness, setArray should probably receive the expression to forward it here.

Reply:
Done.  Thanks for the help!